gae/404.tpl

```diff
@@ -4,7 +4,7 @@
     <title>Page not found</title>
     <meta charset="utf-8">
     <meta name="robots" content="noindex, nofollow">
-    <meta name="viewport" content="initial-scale=1, minimum-scale=1, width=device-width">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0, minimum-scale=1.0">
```

Comment:
To be consistent with DevSite, let's use `<meta name="viewport" content="width=device-width, initial-scale=1">`